src/megaapi_impl.cpp

```diff
@@ -12425,6 +12455,22 @@ void MegaApiImpl::sendPendingRequests()
 
             break;
 		}
+        case MegaRequest::TYPE_GET_USER_EMAIL:
+        {
+            handle uh = request->getNodeHandle();
+            if (uh == INVALID_HANDLE)
+            {
+                e = API_EARGS;
+                break;
+            }
+
+            char uid[12];
+            Base64::btoa((byte*)&uh, sizeof uh, uid);
```

Comment:
I think that the size should be `MegaClient::USERHANDLE` instead of `sizeof uh` despite in this case both are the same.